packages/plugin-ext/src/main/browser/style/tree.css

```
@@ -40,3 +40,7 @@
     margin-bottom: 10px;
     margin-left: 17px;
 }
+
+.theia-TreeNode:not(:hover):not(.theia-mod-selected) .theia-tree-view-inline-action {
```

@msujew should we prefix the entry with `.theia-tree-view` as not to potentially affect other views and keep the styling specific to the plugin-system?

```suggestion
.theia-tree-view .theia-TreeNode:not(:hover):not(.theia-mod-selected) .theia-tree-view-inline-action {
```

---

Sure, sounds like a good idea 👍